backend/src/main/java/com/bakdata/conquery/commands/ManagerNode.java

```diff
@@ -183,8 +183,9 @@ public void run(ConqueryConfig config, Environment environment) throws Interrupt
 	}
 
 	private void loadMetaStorage() {
-		log.info("Started meta storage");
-		storage = new MetaStorage(validator, config.getStorage(), datasetRegistry);
+		log.info("Opening MetaStorage");
+		storage.openStores(config.getStorage());
```

Comment:
Warum musst du das an der Stelle trennen?

Reply:
Damit ich den MetaStorage schon in den OM injecten kann, bevor der MetaStorage sich lädt und sich in die Objecte selbst injected. 
Ich hätte im Constructor vom MetaStorage ein `this.injectInto(OM)` machen können, aber das sollte man soweit ich weiß nicht machen, weil sich `this` noch in der Initialisierung befindet